zpeclient_test.go

```diff
@@ -9,13 +9,37 @@ import (
 
 	"github.com/ardielle/ardielle-go/rdl"
 	"github.com/yahoo/athenz/clients/go/zms"
-	v1 "github.com/yahoo/k8s-athenz-istio-auth/pkg/apis/athenz/v1"
+	v1 "github.com/yahoo/k8s-athenz-syncer/pkg/apis/athenz/v1"
+	"github.com/yahoo/k8s-athenz-syncer/pkg/client/clientset/versioned/fake"
+	athenzInformer "github.com/yahoo/k8s-athenz-syncer/pkg/client/informers/externalversions/athenz/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
 )
 
 const (
-	domainName = "home.domain"
-	username   = "user.name"
+	domainName      = "home.domain"
+	username        = "user.name"
+	trustDomainName = "test.delegated.domain"
+	trustusername   = "trustuser.name"
+)
+
+var (
+	ad = &v1.AthenzDomain{
```

Comment:
Should we use the `getFakeAthenzDomains` and `getFakeTrustAthenzDomains` functions to set ad / ad1?

Reply:
I moved the code piece to NewCache() method to avoid code duplication